readme.md

```diff
@@ -819,6 +820,92 @@ Type: `string`
 
 The IP address used to send the request from.
 
+### HTTPS
+
+##### ca
+
+Type: `string` | `string[]` | `Buffer` | `Buffer[]`
+
+Override the default CAs ([from Mozilla](https://ccadb-public.secure.force.com/mozilla/IncludedCACertificateReport))
+
+```js
+// Single CA
+got('https://example.com', {ca: fs.readFileSync('./my_ca.pem')});
+
+// Multiple CAs
+got('https://example.com', {
+	ca: [
+		fs.readFileSync('./my_ca1.pem'),
+		fs.readFileSync('./my_ca2.pem')
+	]
+});
+```
+
+##### key
+
+Type: `string` | `string[]` | `Buffer` | `Buffer[]` | `Object[]`
+
+Private keys in PEM format.\
+PEM allows the option of private keys being encrypted. Encrypted keys will be decrypted with `options.passphrase`.\
+Multiple keys with different passphrases can be provided as an array of `{pem: <string|Buffer>, passphrase: <string>}`
+
+##### cert
+
+Type: `string` | `string[]` | `Buffer` | `Buffer[]`
+
+Cert chains in PEM format.\
```

Comment:
"Cert"

Use the full word.

Also linkify to a place that explains "certificate chains".

Reply:
Is [wikipedia](https://en.wikipedia.org/wiki/X.509#Certificate_chains_and_cross-certification) fine?

Reply:
MDN is better. Do they have it?

Reply:
I can only find a generic page about certificates
https://developer.mozilla.org/en-US/docs/Mozilla/Security/x509_Certificates
And a small paragraph about Certificate Authorities
https://developer.mozilla.org/en-US/docs/Glossary/Certificate_authority

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Public_Key_Pinning

Reply:
> https://developer.mozilla.org/en-US/docs/Web/HTTP/Public_Key_Pinning

That's about HPKP, that's a browser thing. The Certificate Pinning I'm talking about is nearly the same thing, but it's performed voluntarily by the client (application), the client specify which CAs and/or certificate to trust.

Reply:
Then Wikipedia it is :)